app/assets/stylesheets/widgets/recaptcha.scss

```diff
@@ -0,0 +1,5 @@
+.g-recaptcha {
+  > div {
+    margin: auto;
+  }
+}
```

Comment:
This is to center the recaptcha box. We have no way of making it's width longer, thus centering might be our best bet. Happy to remove this otherwise.